src/index.html

```diff
@@ -7,6 +7,17 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
-  <h1>Quiz App 123 456</h1>
+  <h1>Quiz App</h1>
```

Comment:
Consider wrapping your `h1 `in a `header` element

Reply:
Won't fix